### ISSUE TITLE
./run_tests.py doesn't show error when passing invalid test

### ISSUE THREAD

**Issue by Makman2** [coala-analyzer/coala#384]
Is this really intended?
```shell
./run_tests.py -t MyNotExistingTest
```
outputs:
```

Tests finished: failures in 0 of 0 test modules, skipped 0 test modules.
```

**Comment by sils1297** [comment]
maybe we could add a message "...Test could not be run because it does not exist." special care might be needed for skipped tests so they don't appear.

**Comment by Makman2** [comment]
I think we could even make an error out of it, since this indicates that something went wrong during invocation^^

**Comment by Udayan12167** [comment]
This should be easy i'll take this?

**Comment by sils1297** [comment]
sure :)
